### DISCUSSION TITLE
How to import and use this library in XCode?

### DISCUSSION THREAD

**Discussion by gavrilikhin-d** [siteline/SwiftUI-Introspect#328] [Q&A] [answered]
I have a XCode 15 beta.  I've added this library as a dependency through XCode with everything enabled
<img width="665" alt="image" src="https://github.com/siteline/SwiftUI-Introspect/assets/75179902/f0977145-1961-4ba6-b756-bc20be157752">
However, there is no `.introspect` on `TextField` or any other view.
<img width="391" alt="image" src="https://github.com/siteline/SwiftUI-Introspect/assets/75179902/4678b4ae-ee41-43b6-bb69-442cc6da3cea">
What am I doing wrong?

**Comment by davdroman** [comment]
You're only meant to check this package in:

<img width="665" alt="260292065-f0977145-1961-4ba6-b756-bc20be157752" src="https://github.com/siteline/SwiftUI-Introspect/assets/2538074/a29fdaf6-5291-49aa-b7b6-c87fcd1c9963">

Note Introspect [is a legacy module](https://github.com/siteline/SwiftUI-Introspect#swiftui-introspect) that will be removed next month along with the official iOS 17 release.

**Reply by gavrilikhin-d** [reply]
Still have no `introspect` :(

**Reply by davdroman** [reply, marked as answer]
I assume you did `import SwiftUIIntrospect` as well? If it still doesn’t work, could you please share a small reproducible example?

**Reply by gavrilikhin-d** [reply]
Ops, I forgot it XD 
Adding `import SwiftUIIntrospect` fixed the issue.
Thanks!

**Reply by gavrilikhin-d** [reply]
When are the last 2 `SwiftUIIntrospect` imports needed?

**Reply by davdroman** [reply]
See https://github.com/siteline/SwiftUI-Introspect/pull/168 for details. Very rarely would you need to explicitly link to the dynamic/static versions.
